packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -172,18 +163,12 @@ export const Tr = styled.tr<{
     ${({ $index = 0 }) =>
         $index % 2 === 1
             ? `
-                background-color: ${transparentize(
-                    0.7,
-                    DEFAULT_THEME.colors.gray[1],
-                )};
+                background-color: var(--mantine-color-ldGray-0);
```

Comment:
maybe we should live this intact for light mode and use variable for the dark mode?